### PR TITLE
Writethedocs Vilnius link fixing

### PR DESCRIPTION
**Pull request by ElinaMcC**

### CODE REVIEW

File: docs/conf/eu/2015/schedule.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ The talks will take place at the **Klub Lavka, Novotného lávka 201/1, Praha 1*
   <tr>
     <td class="schedule-time">3:50 PM</td>
     <td><a href="/conf/eu/2015/speakers/#speaker-cward">Documenting your Story - Crafting a good presentation</a> - Chris Ward</td>
-  	<td><a href="http://gregariousmammal.com:8000/good_presentations/#/">Slides</a></td>
+  	<td></td>
```
Comment (plaindocs) [inline docs/conf/eu/2015/schedule.md]:
```suggestion

```

File: docs/conf/eu/2015/schedule.md [hunk 2]
```diff
@@ -221,7 +221,7 @@ The talks will take place at the **Klub Lavka, Novotného lávka 201/1, Praha 1*
   <tr>
     <td class="schedule-time">11:30 AM</td>
     <td><a href="/conf/eu/2015/speakers/#speaker-ecaine">How to Write an Email</a> - Elijah Caine</td>
-  	<td><a href="http://slides.elijahcaine.me/write-the-docs-eu-2015_how-to-write-an-email/">Slides</a></td>
+  	<td></td>
```
Comment (plaindocs) [inline docs/conf/eu/2015/schedule.md]:
```suggestion

```

File: docs/conf/eu/2015/schedule.md [hunk 3]
```diff
@@ -269,7 +269,7 @@ The talks will take place at the **Klub Lavka, Novotného lávka 201/1, Praha 1*
   <tr>
     <td class="schedule-time">3:00 PM</td>
     <td><a href="/conf/eu/2015/speakers/#speaker-ajaruga">The quest for scientific credit for software documentation</a> - Anna Jaruga</td>
-  	<td><a href="http://www.igf.fuw.edu.pl/~ajaruga/talks/Jaruga_WTD_2015.pdf">Slides</a><td>
+  	<td><td>
```
Comment (plaindocs) [inline docs/conf/eu/2015/schedule.md]:
```suggestion

```

**Review by plaindocs — CHANGES_REQUESTED** [review]
:+1:

File: docs/conf/eu/2016/news/day-of.md [hunk 4]
```diff
@@ -15,15 +15,15 @@ all in one place.
...
 * **Come to the party on Monday night.** This is a private party just for conference attendees. Plan to attend so you can get to know folks in the community. We'll be at [Klub Lavka, Novotného lávka 201/1](https://goo.gl/maps/3k5XZQvkHZr), our awesome conference venue from last year! Party time is 19:00-23:00, light snacks and various beverages provided.
 * **Coffee, breakfast, and lunch are provided.** Don't worry about coffee before you come to the conference on Sunday, Monday or Tuesday. Coffee and breakfast, including gluten free and veggie options are served at 09:00, and the conference starts at 10:00.
 
 If you want to keep in touch with us during the conference,
 [Slack](http://slack.writethedocs.org) or [Twitter](http://twitter.com/writethedocs) are the best ways.
 For anything Code of Conduct related,
-we have [personal phone numbers and email](https://www.writethedocs.org/code-of-conduct/#staff-contact-information) listed as well.
+we have [personal phone numbers and email](http://www.writethedocs.org/code-of-conduct/#reporting-and-contact-information) listed as well.
```
Comment (sergeykolosov) [inline docs/conf/eu/2016/news/day-of.md]: Whenever possible, prefer secure links (https). It works for this one.
```suggestion
we have [personal phone numbers and email](https://www.writethedocs.org/code-of-conduct/#reporting-and-contact-information) listed as well.
```

File: docs/conf/na/2016/news/thanks-for-coming.md [hunk 5]
```diff
@@ -34,7 +34,7 @@ The Write the Docs Slack has really taken off this year! Our general Slack chann
 
 ## Meetups
 
-We hope you continue to [build the community](https://www.writethedocs.org/meetups/) and keep in touch with other documentarians in your hometown. In this spirit, we have a number of meetups in cities across the country. If you don't see your city [listed](https://www.writethedocs.org/meetups/), consider [starting](https://www.writethedocs.org/meetups/starting/) a meetup! 
+We hope you continue to [build the community](https://www.writethedocs.org/meetups/) and keep in touch with other documentarians in your hometown. In this spirit, we have a number of meetups in cities across the country. If you don't see your city [listed](https://www.writethedocs.org/meetups/), consider [starting](http://www.writethedocs.org/organizer-guide/meetups/starting/) a meetup!
```
Comment (sergeykolosov) [inline docs/conf/na/2016/news/thanks-for-coming.md]: Whenever possible, prefer secure links (https).
```suggestion
We hope you continue to [build the community](https://www.writethedocs.org/meetups/) and keep in touch with other documentarians in your hometown. In this spirit, we have a number of meetups in cities across the country. If you don't see your city [listed](https://www.writethedocs.org/meetups/), consider [starting](https://www.writethedocs.org/organizer-guide/meetups/starting/) a meetup! 
```

File: docs/blog/newsletter-december-2016.md [hunk 6]
```diff
@@ -26,12 +26,12 @@ One of our most involved discussions this month was about limiting the length of
...
 
 The discussion wrapped up with a 74-word sentence quoted from _Emma_ by Jane Austen:
 > "It was some comfort to him that many inquiries after himself and Miss Woodhouse (for his neighbours knew that he loved to be inquired after), as well as Miss Smith, were coming in during the rest of the day; and he had the pleasure of returning for answer, that they were all very indifferent—which, though not exactly true, for she was perfectly well, and Harriet not much otherwise, Emma would not interfere with."
 
-For those keeping score at home, that includes seven commas, one semi-colon, a parenthetical, and an em dash. If you’re curious, the sentence’s Flesch-Kincaid grade level is 15.4!
+For those keeping score at home, that includes seven commas, one semi-colon, a parenthetical, and an em dash. If you’re curious, the sentence’s Flesch-Kincaid grade level is 15.4!
```
Comment (sergeykolosov) [inline docs/blog/newsletter-december-2016.md]: Seemingly unneeded trailing space.

```suggestion
For those keeping score at home, that includes seven commas, one semi-colon, a parenthetical, and an em dash. If you’re curious, the sentence’s Flesch-Kincaid grade level is 15.4!
```

**Review by sergeykolosov — CHANGES_REQUESTED** [review]

**Review by plaindocs — APPROVED** [review]
Awesome, thank you!

**Pull request merged**